packages/react-components/react-dialog/src/components/DialogActions/DialogActions.types.ts

```diff
@@ -0,0 +1,25 @@
+import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+
+export type DialogActionsSlots = {
+  root: Slot<'div'>;
+};
+
+export type DialogActionsPosition = 'left' | 'right';
```

Comment:
```suggestion
export type DialogActionsPosition = 'start' | 'end';
```

`left` and `right` don't seem consistent with our RTL terms which generally tend to be `start` and `end`